build_misc/entitlements/x8A4.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>get-task-allow</key>
+        <true/>
+        <key>task_for_pid-allow</key>
+        <true/>
+        <key>platform-application</key>
+        <true/>
+        <key>com.apple.private.kernel.get-kext-info</key>
+        <true/>
+        <key>com.apple.private.security.no-container</key>
+        <true/>
+        <key>com.apple.security.iokit-user-client-class</key>
+        <string>AppleMobileApNonceUserClient</string>
+        <key>com.apple.security.exception.iokit-user-client-class</key>
+        <string>IOSurfaceRootUserClient</string>
```

Comment:
As far as I know, these two entitlements are equivalent, any specific reasons to use both variants instead of `com.apple.security.iokit-user-client-class` with an array?